### DISCUSSION TITLE
Connection draining on service update

### DISCUSSION THREAD

**Discussion by VoidMonk** [caprover/caprover#1702] [Q&A]
Hey all,

Is it possible in CapRover via Docker swarm or Nginx to wait and drain pending connections to a running service container before removing it when an update is performed and a new container is created on the nodes?

So, the new container accepts new connections while the old container waits for a configurable number of seconds to drain pending connections before its removed.

**Comment by githubsaturn** [comment]
no this is not supported natively by CapRover.
